### ISSUE TITLE
Escape in window switcher causes window to lose keyboard focus

### ISSUE THREAD

**Issue by inflmts** [labwc/labwc#3650]
### Problem description

When pressing `Escape` in the window switcher, labwc raises the original window but doesn't restore keyboard focus. This happens even when there is only one window. I sometimes used this to preview window contents without switching.

### Steps to reproduce

1. Open one or more windows (eg. foot).
2. Press `A-Tab` and hold `Alt` to open the window switcher.
3. Press `Escape`.

Expected behavior: The window that was active before cycling should have keyboard focus again.
Actual behavior: No windows have keyboard focus.

### labwc build source

Release

### labwc version

0.20.1

### labwc environment

From a TTY or some display manager like lightdm

### Distribution

Void Linux
